src/weights.jl

```diff
@@ -216,37 +220,55 @@ For each element `i` in `t` the weight value is computed as:
   As this value approaches 0, the resulting weights will be almost equal,
   while values closer to 1 will put greater weight on the tail elements of the vector.
 
+# Keyword arguments
+
+- `scaled::Bool`: Whether are not to return the scaled
+
 # Examples
 ```julia-repl
 julia> eweights(1:10, 0.3)
 10-element Weights{Float64,Float64,Array{Float64,1}}:
- 0.3
- 0.42857142857142855
- 0.6122448979591837
- 0.8746355685131197
- 1.249479383590171
- 1.7849705479859588
- 2.549957925694227
- 3.642797036706039
- 5.203995766722913
- 7.434279666747019
-```
-"""
-function eweights(t::AbstractVector{T}, λ::Real) where T<:Integer
+ 0.04035360699999998
+ 0.05764800999999997
+ 0.08235429999999996
+ 0.11764899999999996
+ 0.16806999999999994
+ 0.24009999999999995
+ 0.3429999999999999
+ 0.48999999999999994
+ 0.7
+ 1.0
+
+# Links
+- https://en.wikipedia.org/wiki/Moving_average#Exponential_moving_average
+- https://en.wikipedia.org/wiki/Exponential_smoothing
+ ```
+"""
+function eweights(t::AbstractVector{<:Integer}, λ::Real, n::Integer; scaled::DepBool=nothing)
```

Comment:
Maybe call this `scale`? We already use this term for transformations (https://juliastats.org/StatsBase.jl/latest/transformations/).